docs/modules/transform-sync/how-it-works.md

```diff
@@ -45,9 +36,26 @@ component TransformInternal {
-2. **Save bandwidth.** The `Position` component represents location with 3 doubles. The `TransformInternal` component represents location as 3 floats, which is the Unity native representation. This means that, ignoring rotation, a position update is twice as large as a transform update. If the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate then you have a net bandwidth saving over updating just the `Position` component at a high frequency!
-3. **Atomicity.** The `Transform` component contains more than just the location. If the other fields were on a separate component, you lose the guarantee that all fields are updated atomically.<br/><br/>Two updates sent in the same frame from one worker are _not_ guaranteed to be received on the same frame in another.
+###### **1. Separation of responsibilies.**
+
+The `Improbable.Position` component is the load balancer's representation of location and the `TransformInternal` component is the workers' representation of location. This allows you to abstract the concept of location from the load balancer. A gameplay programmer would only need to be aware of the `TransformInternal` component without having to know about the load balancer at all.
+
+###### **2. Save bandwidth.**
+
+The `TransformInternal` component contains the compressed representations of location, rotation and velocity.
+
+Rotation is compressed from a full 128 bit Quaternion to a 32 bit uint. A `FixedPointVector3` type exists to represent a 3D vector as 3 fixed point values, each of which uses variable-length zig-zag encoding. This means that location and velocity now require _at most_ 96 bits each, instead of _always_ requiring 96 bits.
+
+For comparison, the `Position` component uses 3 doubles, which always takes 192 bits compared to a `FixedPointVector3` using _at most_ 96 bits. This means that, only considering location, a `Position` update is _at least_ twice as large as a comparable `TransformInternal` update.
+
+If the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate, then you have a net bandwidth saving compared to updating just the `Position` component at a high frequency!
+
+> The `TransformInternal` compression scheme comes with a drop in precision. The `FixedPointVector3` type utilises the Q21.10 fixed point format, for a range of -2097152 to 2097151.999 at 0.977mm (2^-10 metres) precision. The `CompressedQuaternion` utilises 10 bits to encode each quaternion element, which translates to a mean precision in Euler angles of 0.08 degrees.
```

Comment:
@jamiebrynes7  would you prefer this to follow on from the 2nd paragraph - and compare the range of a original types to the compressed forms?

Reply:
It might be nice to split these out a little bit? 

Talk about position, talk about `FixedPointVector3`, talk about precision loss.
Talk about rotation, talk about the compression, talk about precision loss.